pkg/tests/cross-tests/tfwrite.go

```diff
@@ -38,6 +39,87 @@ func WriteHCL(out io.Writer, sch map[string]*schema.Schema, resourceType, resour
 	return err
 }
 
+type PFWriter struct {
+	out io.Writer
+}
+
+func WritePF(out io.Writer) PFWriter { return PFWriter{out} }
```

Comment:
Would be good to add some test coverage for this specifically, to make sure nothing unexpected pops up in the bigger tests.